code/game/objects/items.dm

```diff
@@ -972,9 +972,9 @@
 		eyes.damage += rand(3,4)
 		if(eyes.damage >= eyes.min_bruised_damage)
 			if(M.stat != 2)
-				if(eyes.robotic <= 1) //robot eyes bleeding might be a bit silly
+				if(eyes.robotic >= 1) //robot eyes bleeding might be a bit silly
```

Comment:
?

Reply:
Used to be it checks if your robotic level is less than or equal to 1, rather than greater than or equal to 1.

This meant it would never show for regular eyes, but would show for the cybernetic eyes. The inverse of what that comment suggests

Reply:
Isn't it still only going to show for robotic eyes like this, since regular eyes have this var set to 0